benchmarks/asv.conf.json

```diff
@@ -3,29 +3,54 @@
     "project": "scitools-iris",
     "project_url": "https://github.com/SciTools/iris",
     "repo": "..",
-    "environment_type": "conda-delegated",
+    "environment_type": "delegated",
     "show_commit_url": "https://github.com/scitools/iris/commit/",
     "branches": ["upstream/main"],
-    "build_command": [
-        "python setup.py build",
-        "python -mpip wheel --no-deps -w {build_cache_dir} {build_dir}"
-    ],
 
     "benchmark_dir": "./benchmarks",
     "env_dir": ".asv/env",
     "results_dir": ".asv/results",
     "html_dir": ".asv/html",
-    "plugins": [".asv_delegated_conda"],
+    "plugins": [".asv_delegated"],
+
+    "delegated_env_commands_comment": [
+        "The command(s) that create/update an environment correctly for the",
+        "  checked-out commit.",
+        "The commit key indicates the earliest commit where the command(s)",
+        "  will work.",
+        "The first 'command' in a sequence MUST ONLY set the ENV_PARENT",
+        "  env var - the directory within which the environment directory",
+        "  will be located. E.g. `ENV_PARENT=foo/` . Available as",
+        "  {env_parent} in subsequent commands.",
+        "The environment will be detected as the most recently updated",
+        "  environment in {env_parent}.",
+        "Commands are interpreted the same as build_command, with following",
+        "  exceptions:",
+        "  * Env vars limited to those set outside build time.",
+        "    (e.g. {conf_dir} available but {build_dir} not)",
+        "  * Are run in the same environment as the ASV install itself."
+    ],
+    "delegated_env_commands": {
+        "c8a663a0": [
+            "ENV_PARENT={conf_dir}/.asv/env/nox312",
+            "PY_VER=3.12 nox --envdir={env_parent} --session=tests --install-only --no-error-on-external-run --verbose"
+        ],
+        "d58fca7e": [
+            "ENV_PARENT={conf_dir}/.asv/env/nox311",
+            "PY_VER=3.11 nox --envdir={env_parent} --session=tests --install-only --no-error-on-external-run --verbose"
+        ],
+        "44fae030": [
+            "ENV_PARENT={conf_dir}/.asv/env/nox310",
+            "PY_VER=3.10 nox --envdir={env_parent} --session=tests --install-only --no-error-on-external-run --verbose"
+        ]
+    },
 
-    // The command(s) that create/update an environment correctly for the
-    //  checked-out commit.
-    // Interpreted the same as build_command, with following exceptions:
-    //  * No build-time environment variables.
-    //  * Is run in the same environment as the ASV install itself.
-    "delegated_env_commands": [
-        "PY_VER=3.12 nox --envdir={conf_dir}/.asv/env/nox01 --session=tests --install-only --no-error-on-external-run --verbose"
+    "command_comment": [
+        "We know that the Nox command takes care of installation in each ",
+        " environment, and in the case of Iris no specialised uninstall or ",
+        " build commands are needed to get it working."
```

Comment:
It doesn't get parsed anywhere, so nothing to check. I've abandoned the whole indent thing because it seemed unnecessary.

3c7446acb44b0b32e9272bda453cc3e801ff62a2